grakn-test/src/test/java/ai/grakn/test/engine/controller/UserControllerTest.java

```diff
@@ -47,23 +46,8 @@ public void testAddNewUser(){
                 extract().response().andReturn();
 
         //Check he is there
-        assertTrue(users.getUser(userName).is(UsersHandler.USER_NAME, userName));
-    }
-
-    @Test
-    public void testGetUser(){
-        String userName = "bob";
-        String password = "smith";
-
-        //Add User
-        Json user = Json.object(UsersHandler.USER_NAME, userName, UsersHandler.USER_PASSWORD, password);
-        users.addUser(user);
-
-        //Get Him Back
-        Response dataResponse =
-                get("/user/one?" + UsersHandler.USER_NAME + "=" + userName).then().
-                        statusCode(200).extract().response().andReturn();
-
-        assertTrue(dataResponse.getBody().asString().contains(userName));
```

Comment:
To clarify what happened here: I removed the dependency on `UsersHandler`, using the REST API exclusively to add/check for users.

This meant the tests to `GET` and `POST` users became exactly the same! So I deleted one.